src/components/SearchBox.js

```diff
@@ -175,8 +175,9 @@ class SearchBox extends Component {
 
   setValue = ({ value, isOpen = true, ...rest }) => {
     if (this.props.value) {
-      if (this.props.onChange)
+      if (this.props.onChange) {
         this.props.onChange(value, this.triggerQuery, rest.event);
+      }
       return;
```

Comment:
Let's not return instead we can use `else` condition.

Reply:
any specific reasons?